apps/pipeline/upstream/env/aws/secrets-mount.yaml

```diff
@@ -0,0 +1,21 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: kubeflow-secrets-pod
+  namespace: kubeflow
+spec:
+  serviceAccountName: kubeflow-secrets-manager-sa
+  containers:
+  - name: secrets
+    image: nginx:1.14.2
+    volumeMounts:
+    - name: kubeflow-secrets
```

Comment:
Sure can split them into 2 secrets. Initially put them in the same secret because of cost. If you create the secret on the web ui it will automatically populate the secret with port and url, maybe consider copying what they use there for rds secret?